src/sentry/static/sentry/app/components/gridEditable/index.tsx

```diff
@@ -310,6 +311,7 @@ class GridEditable<
       >
         {({hasFeature}) => (
           <React.Fragment>
+            {this.renderDownloadAllButton(hasFeature)}
             {this.renderDownloadCsvButton(hasFeature)}
```

Comment:
I'm not sure we should have two download buttons beside each other. The async download is what people have been asking for. Getting just the current page was a stop-gap solution.

Reply:
One reason to keep the old download might be if we only have 1 page of results, might be worth swapping this button depending on whether there's a new page or not.

Reply:
@wmak That is probably a good call, I'll change the function names and try to implement that

Reply:
Doing a in-place export for single page results is a great idea. If there is more than one page we should offer the async export option.

Reply:
@wmak @markstory Made the change!